src/test/java/com/vishva/Payapp/Service/CreditServiceTest.java

```diff
@@ -35,21 +29,9 @@
 
     @BeforeEach
     void setup() {
-        objectMapper = new ObjectMapper();
         MockitoAnnotations.initMocks(this);
     }
 
-    @Test
-    public void amountCreditedNullRequest() {
-
-        try {
-            creditService.amountCredited(null);
-            fail();
-        } catch (IllegalArgumentException e) {
-            assertNotNull(e);
-            assertNotNull(e.getMessage());
-        }
-    }
 
     @Test
     public void accIdDoesNotExist() {
```

Comment:
Still you haven't corrected this testcase.

Here also you need to stub and verify whether creditRepositoryService getAccountEntities is called or not.

Reply:
Okay